teams.txt

```diff
@@ -4,8 +4,8 @@
 [ GitHub repository link ]
 [ Heroku/Deployment link ]
 
-Jason Paffenback, Anthony Mitchell, Taylor Yao
-One.Beer.Later.
-A revamp of our first project, we plan to add more functionality, user authentication, and new technologies. It will have similar look.
-https://github.com/jpaffenback/One.Beer.Later.
-https://one-beer-later.herokuapp.com/
+Angela Johnson,Sasi Koramutla,Tony Scott
+Viber
```

Comment:
Viber is already an app name. (https://www.viber.com/)

Reply:
Yes, but this for educational purposes only, not looking to get legal rights to the name.

Reply:
We'll name it Vyber instead.

Reply:
Vyber is fine, just make the change here in the PR where necessary.